overlord/snapstate/snapstate.go

```diff
@@ -2928,7 +2944,7 @@ func removeInactiveRevision(st *state.State, name, snapID string, revision snap.
 
 // RemoveMany removes everything from the given list of names.
 // Note that the state must be locked by the caller.
-func RemoveMany(st *state.State, names []string) ([]string, []*state.TaskSet, error) {
+func RemoveMany(st *state.State, names []string, transactional bool) ([]string, []*state.TaskSet, error) {
```

Comment:
As I mentioned in the other comment I think we cannot suport this with RemoveMany.